public/javascripts/ng-apps.js

```diff
@@ -1,17 +1,86 @@
-var springInitiative = angular.module('springInitiative', ['ngRoute']);
+var springInitiative = angular.module('springInitiativeApp', ['ui.router']);
 
-springInitiative.config(function($routeProvider) {
-  $routeProvider
-    .when('/student', {
-      templateUrl: 'views/student.html',
-      controller: 'mainController'
-    })
-    .when('/login', {
-      templateUrl: 'views/login.html',
+springInitiative.config(function($stateProvider, $urlRouterProvider) {
+
+  $urlRouterProvider.otherwise('/login');
+
+  $stateProvider
+
+    .state('login', {
+      url: '/login',
+      templateUrl: 'views/partial-login.html',
       controller: 'loginCtrl'
     })
-    .when('/', {
-      templateUrl: 'views/login.html',
-      controller: 'loginCtrl'
+
+    .state('index', {
+      url: '/',
+      views: {
+        '': {
+            templateUrl: 'views/index.html',
```

Comment:
`main.html` includes everything you'd expect to see in a blank webpage - scripts, the page title, head and body tags, etc. however, the `<body />` tags only hold one div, `<div ui-view />`. this is the equivalent of `<div ng-view />`
`index.html` is code that's injected into `<div ui-view />`. so is `login.html`.
pages such as `student.html` are then injected into `index.html`